### PR TITLE
Add mission statement page

### PR DESCRIPTION
**Pull request by patrickblackjr**
Also includes: chore(prettier): config update to require ;;;

### CODE REVIEW

**Comment by patrickblackjr** [comment]
An overall list of things changed in this PR:

- Fixed the ESLint and Prettier configurations so they aren't fighting each other. I'm gonna submit another PR for a README update on how to get your VS Code to function correctly and automatically comply with linting.
- Added mission statement

This commit should definitely be squashed, and would probably constitute a version bump (probably minor if we are following semantic versioning).

File: .eslintrc.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    'plugin:vue/recommended',
+    'eslint:recommended',
+    'prettier/vue',
+    'plugin:prettier/recommended'
   ],
-  // add your custom rules here
   rules: {
-    'no-console': 0
+    'vue/component-name-in-template-casing': ['error', 'PascalCase'],
+    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
```
Comment (westwick) [inline .eslintrc.js]: good idea

Comment (patrickblackjr) [reply .eslintrc.js]: 😄

File: pages/mission.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      missionStatementPart1: `Undeclared Development is an international development 
+      group formed from a shared desire of self improvement. Our membership is 
+      comprised of web developers of varied backgrounds ranging from novice coders 
+      to seasoned veterans.`,
```
Comment (westwick) [inline pages/mission.vue]: I'd just leave out the `<script />` altogether and just put the text right in the template, unless this is gonna get fancier, but that's just me, wouldn't hold merging over it!

Comment (patrickblackjr) [reply pages/mission.vue]: Alright I'll merge. I was planning to move it to an external component at some point, so I figure I'd save me the trouble of moving it to the `data()` function now. I am planning on updating styling as well, it's a bit iffy.

**Pull request merged**